src/module/apps/sidebar/encounter-tracker.ts

```diff
@@ -63,11 +63,18 @@ export class EncounterTrackerPF2e<TEncounter extends EncounterPF2e | null> exten
     /** Make the combatants sortable */
     override activateListeners($html: JQuery): void {
         const html = $html[0];
-        const tracker = htmlQuery(html, "#combat-tracker");
+        let tracker = htmlQuery(html, "#combat-tracker");
         if (!tracker) throw ErrorPF2e("No tracker found");
 
         const encounter = this.viewed;
         if (!encounter) return super.activateListeners($html);
+        const trackerParent = tracker.parentElement;
+        const trackerPlaceholder = document.createElement("div");
+
+        if (trackerParent) {
+            trackerParent.insertBefore(trackerPlaceholder, tracker);
+            tracker = trackerParent.removeChild(tracker);
```

Comment:
Is this not referentially identical to the variable's existing value?

Reply:
dang, yeah it is.